keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/NotifierRepository.kt

```diff
@@ -0,0 +1,39 @@
+package com.netflix.spinnaker.keel.persistence
+
+import com.netflix.spinnaker.keel.notifications.NotificationScope
+import com.netflix.spinnaker.keel.notifications.Notifier
+import org.springframework.beans.factory.annotation.Value
+import java.time.Clock
+
+/**
+ * A repository for storing a list of ongoing notifications, and calculating whether
+ * they should be resent
+ */
+abstract class NotifierRepository(
```

Comment:
As I pointed out in the previous PR, I think we should fix the discrepancy between the terminology and comments and the class names. If we're storing a "list of ongoing notifications", then this should be a `NotificationRepository`.

Reply:
Okay, I can change it to be `NotificationRepository`